src/ensembl/src/styles/_mixins.scss

```diff
@@ -0,0 +1,16 @@
+@use "sass:map";
+
+$lozengeDefaults: (
+  "border-radius": 20px,
+  "padding": 6px 20px,
+  "line-height": 1
+);
+
+@mixin lozenge($lineHeight: map-get($lozengeDefaults, "line-height"), $borderRadius: map-get($lozengeDefaults, "border-radius")) {
+  display: inline-block;
+  padding: map-get($lozengeDefaults, "padding");
+  border-radius: $borderRadius;
+  line-height: $lineHeight;
+  user-select: none;
+  margin-bottom: 4px;
+}
```

Comment:
Have you tried optional arguments and keyword arguments (https://sass-lang.com/documentation/at-rules/mixin)

E.g. why would we even need a map here? Isn't this simpler?

```
lozenge($lineHeight: 1, $borderRadius: 20px, $padding: 6px 20px) {}
```

Also, from the consumer perspective (`RoundButton.scss`), wouldn't named arguments be better:

```
@include lozenge($borderRadius: 15px);
```

hopefully this should help us avoid passing the same arguments as the default ones.

Reply:
yes true. Thanks.